pkg/autoscaler/statforwarder/forwarder.go

```diff
@@ -301,7 +303,7 @@ func (f *Forwarder) setProcessor(bkt string, p *bucketProcessor) {
 	f.processors[bkt] = p
 }
 
-func (f *Forwarder) createProcessor(bkt, holder string) *bucketProcessor {
+func (f *Forwarder) createProcessor(ns, bkt, holder string) *bucketProcessor {
```

Comment:
If we're not in direct pod mode. Do we need to redial? Since the server connection will be killed anyway (either because the pod died or because the server will reject connection to itself)?
I think a lot of code below is not taking this into consideration?

Also if `podAddressable` is false — it is sticky, so probably we no longer need to even try to create pod connection?

Reply:
Done. PTAL